webrtc-stats.html

```diff
@@ -1224,8 +1222,8 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Count the total number of Negative ACKnowledgement (NACK) packets sent by this
-                  receiver. Calculated as defined in [[!RFC4585]] section 6.2.1.
+                  Count the total number of Negative ACKnowledgement (NACK) packets as defined in [[!RFC4585]]
+                  section 6.2.1 sent by this receiver.
```

Comment:
this reads a bit weird, "sent by this receiver" is at the end. Maybe needs a comma somewhere.